### DISCUSSION TITLE
Maximum concurrent jobs per server

### DISCUSSION THREAD

**Discussion by corradofiore** [jhuckaby/Cronicle#513] [Q&A]
Hello,

I woud like to set up Cronicle as a distributed scheduler (one master, one backup and N workers).  Our scheduled scripts use roughly half a vCPU when they run, therefore I would like to assign a per-server concurrency limit (e.g. 4 vCPU on worker "A" => 8 maximum parallel executions).  From the docs, however, it looks like this is not something that Cronicle implements at the moment, is that correct?

Thank you,
Corrado

**Comment by mcfetz** [comment]
Hi Corrado,

You could clone your script and assign the maximum concurrency to 2 different categories.

Event 1 > Category A > Worker A
Event 2 (cloned Event 1) > Category B > Worker B

Daniel
